crypto/pkcs11key/key.go

```diff
@@ -76,11 +111,15 @@ type Key struct {
 	pin string
 
 	// The public key corresponding to the private key.
-	publicKey rsa.PublicKey
+	rsaPublicKey rsa.PublicKey
```

Comment:
Thanks for the tip, I've been banging my head against the wall on this one. I'll commit the change soon.
If it wasn't for the email notification, this comment would have been buried in the thread.